bin/sidecli.re

```diff
@@ -183,33 +183,30 @@ let create_transaction = {
   let address_from = {
     let doc = "The sending address, or a path to a wallet. If a bare sending address is provided, the corresponding wallet is assumed to be in the working directory.";
     let env = Arg.env_var("SENDER", ~doc);
-    let doc = "The message to print.";
```

Comment:
@EduardoRFS I wonder why Dune didn't complain of shadowing here. Are we missing flags? Is it because of `-p` in the esy build command?

Reply:
I mean, of course, before this PR

Reply:
(please free to mark this thread resolved once you see it. Not a blocker)

Reply:
I think (but I'm not sure) that -p causes dune to build only what's needed for the packages specified in sidechain.install, and sidecli and genesis_helper should be in there, so I don't think that should be the problem. I also wonder if merging #39 would affect this.

Reply:
@callistonianembrace because it is used, we're not doing anything wrong

Reply:
resolving this, as it doesn't seems related to the PR